pkg/flow/reconciler/xmltojsontransformation/controller.go

```diff
@@ -56,7 +58,11 @@ func NewController(
 	}
 
 	impl := reconcilerv1alpha1.NewImpl(ctx, r)
-	targetInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
+	logging.FromContext(ctx).Info("Setting up event handlers")
```

Comment:
Logging this feels very superfluous.
If we start doing that in every component, expect a flood of unnecessary log entries on every startup.

Reply:
removed

Reply:
Followup issue created to remove them where they are https://github.com/triggermesh/triggermesh/issues/479